### ISSUE TITLE
All commands should preserve header metadata

### ISSUE THREAD

**Issue by matentzn** [mapping-commons/sssom-py#23]

**Comment by cmungall** [comment]
yes, a few of the commands lose header info when we convert to a dataframe. We should do some refactoring and pass around a wrapper object that holds a dataframe and a MappingSet object

**Comment by matentzn** [comment]
What applies to this ticket applies to all methods in sssom-py

**Comment by hrshdhgd** [comment]
I'm thinking adding code into `parse_file` which writes another tsv named (say) `headers.tsv` which would have 2 columns. I'm making a naive assumption that most file headers will look like the [basic.tsv](https://github.com/mapping-commons/sssom-py/blob/master/tests/data/basic.tsv) ?

```
#license: "https://creativecommons.org/publicdomain/zero/1.0/"
#mapping_tool: "https://github.com/cmungall/rdf_matcher"
#creator_id: "cjm"
#mapping_date: "2020-05-30"
#curie_map:
#  a: "http://example.org/a/"
#  rdfs: "http://www.w3.org/2000/01/rdf-schema#"
#  x: "http://example.org/x/"
#  y: "http://example.org/y/"
#  z: "http://example.org/z/"
```

So basically `line.split(':')` would divide it into ["license", "https://creativecommons.org/publicdomain/zero/1.0/"] ...so on and so forth forming 2 columns.

Thoughts?

**Comment by matentzn** [comment]
No, this ticket is solved by doing the same as everywhere else: using mapping set data frame. The header information is stored in there - no additional files needed! If you need to create a new msdf you can just copy the header information over.

**Comment by matentzn** [comment]
This should have been fixed by #50 So I will close. In case there are still issues, we will make new tickets
